iceoryx_hoofs/include/iceoryx_hoofs/memory/relative_pointer.inl

```diff
@@ -239,11 +230,71 @@ inline typename RelativePointer<T>::ptr_t RelativePointer<T>::computeRawPtr() co
     return getPtr(segment_id_t{m_id}, m_offset);
 }
 
+// AXIVION Next Construct AutosarC++19_03-A15.5.3, AutosarC++19_03-A15.4.2, FaultDetection-NoexceptViolations : False
+// positive, std::terminate is not called in the c'tor of PointerRepository and noexcept-specification is not violated
```

Comment:
It may not be able to prove that no exception is thrown. Note that the code for static initialization generated by a compiler is not trivial and may involve locks of some kind (depends on type). I am not sure whether this code can throw but I do not think so. 

So I do not know how the analysis concludes that exceptions may be thrown. Static analysis of throw specifications is hard ...

Maybe disable the rule, depending on how many cases there are where it emits false positives.

Reply:
@MatthiasKillat I agree, it's a bit of a weird false positive. Let's monitor how Axivion behaves and then do a decision on the rule.

cc @elBoberido